.circleci/config.yml

```diff
@@ -527,6 +608,13 @@ workflows:
             branches:
               only: master
 
+      - docker_system_tests:
```

Comment:
I expect this to be removed before merge.

Reply:
cleanup in 41feeaf